rules/linux/defense_evasion_hidden_file_dir_tmp.toml

```diff
@@ -30,7 +30,7 @@ tags = ["Elastic", "Linux"]
 type = "query"
 
 query = '''
-event.category:process AND event.type:(start or process_started) AND
+event.category:process AND event.type:(start or process_started or info) AND
```

Comment:
`info` for a process event?
I think we should double check that this is a correct implementation for ECS before we add it to the rule.

Reply:
that the default for process already running state and the main purpose of this PR, not sure if it should be as per [ECS ](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-type.html#ecs-event-type-info) another value ? 
![image](https://user-images.githubusercontent.com/64742097/97198459-1a20b980-17af-11eb-9a36-57657587d32c.png)